docs/💖 Free Training/Alchemy's Road to Web3.md

```diff
@@ -0,0 +1,15 @@
+Learn about [Alchemy's Road to Web3 series](https://docs.alchemy.com/docs/welcome-to-the-road-to-web3), a 10-week, self-paced web3 developer program to help new blockchain devs go from beginner to advanced.
+
+## Road to Web3 Learning Challenges
+Here are links to each of the learning challenges in the Road to Web 3 Series.
+
+[How to Develop an NFT Smart Contract (ERC721) with Alchemy](https://docs.alchemy.com/docs/1-how-to-develop-an-nft-smart-contract-erc721-with-alchemy)
+[How to Build a "Buy Me a Coffee" DeFi dApp](https://docs.alchemy.com/docs/2-how-to-build-buy-me-a-coffee-defi-dapp)
+[How to Make NFTs with On-chain Metadata with Hardhat and Javascript](https://docs.alchemy.com/docs/3-how-to-make-nfts-with-on-chain-metadata-hardhat-and-javascript)
+[How to Create an NFT Gallery](https://docs.alchemy.com/docs/5-connect-apis-to-your-smart-contracts-using-chainlink)
+[How to Create a Dynamic NFT](https://docs.alchemy.com/docs/5-connect-apis-to-your-smart-contracts-using-chainlink)
+[How to Build a Staking dApp](https://docs.alchemy.com/docs/6-how-to-build-a-staking-dapp)
+[How to Build an NFT Marketplace from Scratch](https://docs.alchemy.com/docs/7-how-to-build-an-nft-marketplace-from-scratch)
+[How to Build a Betting Game on Optimism](https://docs.alchemy.com/docs/8-how-to-build-a-betting-game-on-optimism)
+[How to Build a Token Swap Dapp with 0x API](https://docs.alchemy.com/docs/9-how-to-build-a-token-swap-dapp-with-0x-api)
+[How to Create a Decentralized Twitter with Lens Protocol](https://docs.alchemy.com/docs/how-to-create-a-decentralized-twitter-with-lens-protocol)
```

Comment:
What if you make this a list by using `-`?
```suggestion
- [How to Develop an NFT Smart Contract (ERC721) with Alchemy](https://docs.alchemy.com/docs/1-how-to-develop-an-nft-smart-contract-erc721-with-alchemy)
- [How to Build a "Buy Me a Coffee" DeFi dApp](https://docs.alchemy.com/docs/2-how-to-build-buy-me-a-coffee-defi-dapp)
- [How to Make NFTs with On-chain Metadata with Hardhat and Javascript](https://docs.alchemy.com/docs/3-how-to-make-nfts-with-on-chain-metadata-hardhat-and-javascript)
- [How to Create an NFT Gallery](https://docs.alchemy.com/docs/5-connect-apis-to-your-smart-contracts-using-chainlink)
- [How to Create a Dynamic NFT](https://docs.alchemy.com/docs/5-connect-apis-to-your-smart-contracts-using-chainlink)
- [How to Build a Staking dApp](https://docs.alchemy.com/docs/6-how-to-build-a-staking-dapp)
- [How to Build an NFT Marketplace from Scratch](https://docs.alchemy.com/docs/7-how-to-build-an-nft-marketplace-from-scratch)
- [How to Build a Betting Game on Optimism](https://docs.alchemy.com/docs/8-how-to-build-a-betting-game-on-optimism)
- [How to Build a Token Swap Dapp with 0x API](https://docs.alchemy.com/docs/9-how-to-build-a-token-swap-dapp-with-0x-api)
- [How to Create a Decentralized Twitter with Lens Protocol](https://docs.alchemy.com/docs/how-to-create-a-decentralized-twitter-with-lens-protocol)
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant